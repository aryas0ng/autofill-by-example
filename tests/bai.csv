164.98,88.49,374.82
188.03,103.76,391.92
179.28,93.99,361.60
173.95,97.08,398.93
156.24,93.98,448.51
156.24,118.52,
152.32,99.87,
184.65,89.42,
174.04,108.23,
178.32,87.79,
150.82,102.09,
188.8,80.4,
183.3,86.72,
158.49,101.97,
157.27,107.38,
157.34,101.71,
162.17,98.84,
170.99,96.99,
167.28,85.21,
161.65,92.8,
174.47,95.39,
155.58,110.57,
161.69,103.44,
164.65,82.37,
168.24,103.24,
181.41,96.15,
157.99,93.23,
170.57,106.12,
173.7,110.31,
151.86,109.31,