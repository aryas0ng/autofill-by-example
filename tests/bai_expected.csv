164.98,88.49,374.82
188.03,103.76,391.92
179.28,93.99,361.60
173.95,97.08,398.93
156.24,93.98,448.51
156.24,118.52,642.69
152.32,99.87,512.91
184.65,89.42,319.00
174.04,108.23,472.40
178.32,87.79,327.44
150.82,102.09,538.91
188.8,80.4,259.9
183.3,86.72,307.413230533708
158.49,101.97,498.07
157.27,107.38,546.18
157.34,101.71,501.74
162.17,98.84,457.82
170.99,96.99,409.20
167.28,85.21,345.55
161.65,92.8,416.969094924062
174.47,95.39,386.27
155.58,110.57,581.14
161.69,103.44,493.69
164.65,82.37,335.84
168.24,103.24,462.70
181.41,96.15,367.86
157.99,93.23,435.30
170.57,106.12,472.73
173.7,110.31,488.083990722806
151.86,109.31,592.70